inst/csv/replacementPatterns.csv

```diff
@@ -485,22 +485,38 @@ netezza,LOG10(@expression),LOG(@expression)
 netezza,"ISNULL(@a,@b)","COALESCE(@a,@b)"
 netezza,COUNT_BIG(@a),COUNT(@a)
 netezza,USE @schema;,SET search_path TO @schema;
+netezza,"IF OBJECT_ID('@table', 'U') IS NULL CREATE TABLE @table (@definition);",CREATE TABLE IF NOT EXISTS @table (@definition);
 netezza,"IF OBJECT_ID('@table', 'U') IS NOT NULL DROP TABLE @table;",DROP TABLE @table IF EXISTS;
 netezza,.dbo.,.
 netezza,CREATE TABLE #@table (@definition),CREATE TEMP TABLE @table (@definition)
 netezza,CREATE CLUSTERED INDEX @index_name ON @table (@variable);,CREATE INDEX @index_name ON @table (@variable);\nCLUSTER @table USING @index_name;
 netezza,CREATE UNIQUE CLUSTERED INDEX @index_name ON @table (@variable);,CREATE UNIQUE INDEX @index_name ON @table (@variable);\nCLUSTER @table USING @index_name;
 netezza,PRIMARY KEY NONCLUSTERED,PRIMARY KEY
 netezza,VARCHAR(MAX),VARCHAR(1000)
-netezza,FLOAT,NUMERIC
+netezza,FLOAT,"FLOAT(6)"
 netezza,WITH @a AS @b SELECT @c INTO #@d FROM @e;,CREATE TEMP TABLE @d\nAS\nWITH @a AS @b SELECT\n@c\nFROM\n@e;
 netezza,WITH @a AS @b SELECT @c INTO @d FROM @e;,CREATE TABLE @d\nAS\nWITH @a AS @b SELECT\n@c\nFROM\n@e;
 netezza,SELECT @a INTO #@b FROM @c;,CREATE TEMP TABLE @b\nAS\nSELECT\n@a\nFROM\n@c;
 netezza,SELECT @a INTO @b FROM @c;,CREATE TABLE @b AS\nSELECT\n@a\nFROM\n@c;
 netezza,#,
-netezza,"LEFT(@variable,@b)","STRLEFT(@variable,@b)"
-netezza,"RIGHT(@variable,@b)","STRRIGHT(@variable,@b)"
+netezza,"LEFT(@variable,@b)","SUBSTR(@variable, 0, @b+1)"
+netezza,"RIGHT(@variable,@b)","SUBSTR(@variable, length(@variable)-@b+1, @b)"
 netezza,SELECT TOP @([0-9]+)rows @a;,SELECT @a LIMIT @rows;
+netezza,"CONCAT(CONCAT(CONCAT(CONCAT(CONCAT(CONCAT(@a, @b), @c), @d), @e), @f), @g)", @a || @c || @e || @g
+netezza,"CONCAT(CONCAT(CONCAT(CONCAT(CONCAT(CONCAT(CONCAT(CONCAT(@a, @b), @c), @d), @e), @f), @g), @h), @i)", @a || @c || @e || @g || @i
+netezza,"CONCAT(CONCAT(@a, @b), '@c')","@a || @b || '@c'"
+netezza,"CONCAT(CONCAT(@a, '@b'), '@c')","@a || '@b' || '@c'"
+netezza,"CONCAT('@a', @b,'@c',@d)","'@a' || @b || '@c' || @d"
+netezza,"CONCAT(@a, @b)","@a || @b"
+netezza,"CONCAT(@a, '@b')","@a || '@b'"

```

Comment:
This rule is equivalent to the rule above. The wildcards (e.g. `@b`) capture everything, including the single quotes.